src/test/java/org/web3j/gradle/plugin/Web3jPluginTest.java

```diff
@@ -56,6 +60,9 @@ public void generateContractWrappers() throws IOException {
                 "        }\n" +
                 "    }\n" +
                 "}\n" +
+                "dependencies {\n" +
+                "   compile 'org.web3j:core:" + projectVersion + "'\n" +
```

Comment:
We need this in the test ? The plugin class already adds this dependency so user project shouldn't need to explicitly add it.

Reply:
It wouldn't work without it, as it was complaining at runtime that no version was set.

So whilst the dependency is there it couldn't figure out what version, and hence came up as 'unspecified'. 

Is there another way we can specify it?

Reply:
It should be working since the plugin class already adds the web3j dependency:

https://github.com/web3j/web3j-gradle-plugin/blob/master/src/main/java/org/web3j/gradle/plugin/Web3jPlugin.java#L34

The issue comes because web3j version is taken from `target.getVersion()`, but this will not match since the target project has its own version, which in the test is `undefined`.